src/Tests/Navbar.test.js

```diff
@@ -0,0 +1,13 @@
+import { render, screen } from "@testing-library/react";
+import "@testing-library/jest-dom";
+import { MemoryRouter } from "react-router-dom";
+import Navbar from "../components/Navbar";
+
+describe("test for Navbar", () => {
+  test("test to return particular string", () => {
+    render(<Navbar />, {
+      wrapper: MemoryRouter,
+    });
+    expect(screen.getByText("European Countries")).toBeInTheDocument();
+  });
```

Comment:
- [ ] You've done well in writing snapshot tests. As of now we only have two tests in the App and they are all snapshot tests. Besides this, it would be nice to include unit tests for all redux pure functions, and also integration tests using the React Testing Library. Please consider doing this just as required of us in the project's requirements below:

---

> ### Testing requirements
> 
> - Create unit tests for pure functions in your Redux code.
> - Create integration tests for your application using the React Testing Library.
>    - You may need to mock the access to the API, so that your tests don't send actual requests.
>    - You may need to mock the connection to the Redux Store.

---